### ISSUE TITLE
Missing function `lookupTranslationObject`

### ISSUE THREAD

**Issue by jalandis** [ember-intl/ember-intl#1404]
### Environment

- Ember Version: 3.20.3
- Ember CLI Version: 3.20.0
- Ember Intl Version: 5.4.1
- Browser(s): Chrome
- Node Version: 12

### Steps to Reproduce

After attempting to upgrade, all Ember acceptance tests are failing. 

```
this._translationContainer.lookupTranslationObject is not a function
```

**Comment by jasonmit** [comment]
This will be patched in v5.4.1 - thanks for pointing it out!

**Comment by jasonmit** [comment]
@jalandis While fixing it to match 4.x's API I realized I should not be exposing the internal translation models.  I'm likely going to change the behavior of this method in 6.x so that it returns only the translations map (key/original translation string) instead of the internal object.  Basically future API will return the result of todays API: `intlService.translationsFor('en-us').translations`.

Curious, is this how you use `translationsFor()` or do you have another use-case?
